src/js/msp/MSPHelper.js

```diff
@@ -2725,13 +2715,12 @@ MspHelper.prototype.serialPortFunctionMaskToFunctions = function(functionMask) {
 }
 
 MspHelper.prototype.serialPortFunctionsToMask = function(functions) {
-    var self = this;
-    var mask = 0;
+    const self = this;
+    let mask = 0;
 
-    var keys = Object.keys(self.SERIAL_PORT_FUNCTIONS);
-    for (var index = 0; index < functions.length; index++) {
-        var key = functions[index];
-        var bitIndex = self.SERIAL_PORT_FUNCTIONS[key];
+    for (const index in functions) {
```

Comment:
One more time: It is incorrect to replace `for ( ;; )` with `for ( in )`. Please stop doing this.